vital/tests/CMakeLists.txt

```diff
@@ -40,3 +40,4 @@ kwiver_discover_gtests(vital  geodesy     LIBRARIES ${test_libraries})
 kwiver_discover_gtests(vital  geo_point   LIBRARIES ${test_libraries})
 kwiver_discover_gtests(vital  geo_polygon LIBRARIES ${test_libraries})
 kwiver_discover_gtests(vital  track_set   LIBRARIES ${test_libraries})
+kwiver_discover_gtests(vital  image_container_set LIBRARIES ${test_libraries})
```

Comment:
This... well, I'll just note that this is a reason to rebase after #345 lands and re-review the changes in this PR then :smile:. (Namely, this appears to come from an old version of #345...)